src/code_execution/godbolt.rs

```diff
@@ -281,6 +458,42 @@ fn strip_llvm_mca_result(text: &str) -> &str {
     text[..text.find("Instruction Info").unwrap_or_else(|| text.len())].trim()
 }
 
+/// Lists all available godbolt rustc targets
+#[poise::command(prefix_command, broadcast_typing)]
+pub async fn godbolt_targets(ctx: PrefixContext<'_>) -> Result<(), Error> {
+    let mut conn = match update_godbolt_targets(ctx.data).await {
+        Ok(conn) => conn,
+        Err(error) => {
+            log::error!("failed to update godbolt targets list: {:?}", error);
+            ctx.data.database.acquire().await?
+        }
+    };
+
+    let mut targets = sqlx::query!("SELECT name, semver, instruction_set FROM godbolt_targets")
+        .fetch_all(&mut conn)
+        .await?;
+    targets.sort_unstable_by(|lhs, rhs| lhs.semver.cmp(&rhs.semver));
+
+    ctx.msg
```

Comment:
`poise::send_reply(ctx, |msg| ...)` or `ctx.send(|msg| ...)` is a shorter way to write it [and more idiomatic in the discord framework used by rustbot ;)]

That way, godbolt_targets could also be made a slash command